application/src/main/java/org/togetherjava/tjbot/commands/mathcommands/wolframalpha/Constants.java

```diff
@@ -0,0 +1,65 @@
+package org.togetherjava.tjbot.commands.mathcommands.wolframalpha;
+
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+import net.dv8tion.jda.api.requests.restaction.WebhookMessageUpdateAction;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.awt.*;
+import java.awt.font.TextAttribute;
+import java.awt.image.ImageObserver;
+import java.net.http.HttpClient;
+import java.util.Map;
+
+public enum Constants {
+    ;
+
+    /**
+     * Starting part of a regular wolframalpha query link.
+     */
+    public static final String USER_ENDPOINT = "https://www.wolframalpha.com/input";
+    static final Logger logger = LoggerFactory.getLogger(WolframAlphaCommand.class);
+    /**
+     * Maximum Embeds that can be sent in a {@link WebhookMessageUpdateAction}
+     */
+    static final int MAX_EMBEDS = 10;
```

Comment:
https://ci.dv8tion.net/job/JDA5/javadoc/net/dv8tion/jda/api/entities/Message.html#MAX_EMBED_COUNT

Added on latest JDA! Think we have to update though? Could you check for me.

I'll spend some time this weekend / Friday on the bot again to update & check/work on the command system

Reply:
thanks, ill use that constant. theres still quite some stuff that has to be reworked on the logic in this code though. and after the logic works, it has to be beautified heavily. the way it deals with the action hook and all the side effects are quite nasty. but step by step.

Reply:
Yeah, not available on our current JDA build yet. Which version adds it?

Reply:
10, we might aswell update to 11 instantly though.
11 adds support for modals (iirc there's no breaking chances)